lib/processCss.js

```diff
@@ -168,18 +179,16 @@ module.exports = function processCss(inputSource, inputMap, options, callback) {
 					}
 				}
 				return url;
-			}
-		}),
-		extractImports(),
-		modulesScope({
+			},
 			generateScopedName: function generateScopedName (exportName) {
 				return customGetLocalIdent(options.loaderContext, localIdentName, exportName, {
 					regExp: localIdentRegExp,
 					hashPrefix: query.hashPrefix || "",
 					context: context
 				});
-			}
+			},
 		}),
+		icssComposes,
 		parserPlugin(parserOptions)
```

Comment:
I think I've covered a bit of that above, but yeah, I'm not sure about everything.